src/Agent.Worker/TestResults/SimpleTimer.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.TestResults
+{
+    /// <summary>
+    /// This is a utitily class used for recording timing
+    /// information in the verbose trace. Its usage is 
+    /// using (SimpleTimer timer = new SimpleTimer("MyOperation"))
+    ///  {
+    ///     MyOperation...
+    ///  }
+    ///  A   message is recorded in the verbose trace with the time taken
+    ///  for myoperation.
+    /// </summary>
+    internal class SimpleTimer : IDisposable
```

Comment:
do you really need this, please move this to utils folder if really needed.

Reply:
This was done to make agent and library code in parity for verbose logging

Reply:
changing this to be in utils